widget/embedded/src/components/BlockchainList/BlockchainList.tsx

```diff
@@ -48,6 +48,9 @@ export function BlockchainList(props: PropTypes) {
         {blockchains.map((item) => (
           <ListItemButton
             key={`${item.name}-${item.chainId}`}
+            id={`widget-blockchain-list-${item.name
+              .toLowerCase()
+              .replace(/\s+/g, '-')}-item-btn`}
```

Comment:
This code is duplicated in several places. I suggest creating a helper function and using it instead.

Reply:
The **`id`** is incorrect for the list, I use **`className`** instead of id.